docs/guides/migrating-from-camunda-7/migration-tooling/index.md

```diff
@@ -0,0 +1,24 @@
+---
+id: index
+title: Migration tooling
+sidebar_label: Migration tooling
+description: "Understand the available migration tooling and its roadmap."
+---
+
+Camunda currently invests in tooling to help support and ease your migration from Camunda 7 to Camunda 8.
+
+## Migration tools
+
+Camunda provides the following migration tools:
+
+| Migration tool                                                       | Description                                                                                                                                                                  | GitHub link                                                                                                                                                          |
+| :------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **[Migration Analyzer & Diagram Converter](./diagram-converter.md)** | Gain a first understanding of migration tasks. Available for local installation (Java or Docker) or [hosted as a free SaaS offering](https://diagram-converter.camunda.io/). | [Camunda 7 to 8 migration tooling – Migration Analyzer & Diagram Converter](https://github.com/camunda/camunda-7-to-8-migration-tooling/tree/main/diagram-converter) |
```

Comment:
Do we need to rename all instances of this to "Diagram Converter" like we did [here](https://github.com/camunda/camunda-docs/pull/7565/files#diff-bcf510f941b14e0e7f6241be23c53ca2e83796e88cd7b8e877891bebb9b18870R14)?